src/host.ts

```diff
@@ -0,0 +1,41 @@
+import { HostGetter, SingleSpaAureliaOptions } from './types';
+
+export function getHost(hostGetter: HostGetter): never | HTMLElement {
+  const host: HTMLElement = hostGetter();
+
+  if (host) {
+    return host;
+  }
+
+  throw new Error('single-spa-aurelia: hostGetter did not return a valid DOM element');
+}
+
+export function chooseHostGetter(options: SingleSpaAureliaOptions, props: any): HostGetter {
+  props = props?.customProps ?? props;
+
+  if (props.host) {
```

Comment:
This is different from the single-spa convention and breaks things for single-spa parcels. We should this to be called `domElement` and `domElementGetter`. Here are some links to why this would break single-spa parcels:

https://single-spa.js.org/docs/parcels-api#parcel-props

https://github.com/single-spa/single-spa/blob/a2723fabf0d1ef7ca59914236defa9006b6f4531/src/parcels/mount-parcel.js#L65

Reply:
Gotcha, will change it.